htdocs/index.php

```diff
@@ -37,6 +37,8 @@
 
 $hookmanager->initHooks(array('index'));
 
+$langs->load('errors');
```

Comment:
Loading of error file should be done only on error case.
the lang file error should not be loaded in common usage. What is the key missing ?

Reply:
Key translation "WarningLockFileDoesNotExists" is not loading. I found this key in errors.lang file

Reply:
So add the $langs->load('errors');
just before the line that use the key translation (so file will be loaded only when required instead of at each call of page).

Reply:
After looking back on the subject, I notice that  $langs->load('errors') juste before the line that use the key translation is already here. 
I no longer reproduce the problem. I close PR and note my comments on issue.